### PR TITLE
Blog post - Why Lanetix has the best Engineering team

### PR DESCRIPTION
**Pull request by OronNadiv**
@lanetix/engineers BR Please.  Thanks.

### CODE REVIEW

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+3. At Lanetix, we use only the best and newest engineering tool.  For example, we use [NodeJS](http://nodejs.org/), [Java](https://www.java.com/en/), [Bootstrap](http://getbootstrap.com/), [D3](http://d3js.org/), [AngularJS](https://angularjs.org/), [PostgreSQL](http://www.postgresql.org/), [Redis](http://redis.io/), [Elasticsearch](http://www.elasticsearch.org/) and [RabbitMQ](http://www.rabbitmq.com/).  We deploy our servers on [Heroku](https://www.heroku.com/)’s platform, and use [Cloudflare](https://www.cloudflare.com/) to manage our DNS.  We talk to our users using [Intercom](https://www.intercom.io/), and utilize [Pagerduty](http://www.pagerduty.com/) to make sure our servers are always up and running.  We track user errors using [Sentry](https://getsentry.com/), and store our code on [GitHub](https://github.com/).  We use [CircleCI](https://circleci.com/) andfor continuous integration and push our features and bug fixes to production up to several hundreds times per day.  Our skills and tools ensure that the company will move faster than our competitor, and provide the best experience and quality for our users.
```
Comment (apolishch) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: tools not tool

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+7. Our team works hard and plays hard!  Most of the members in our engineering teams are passionate about video games, and spend lunch breaks shooting arrows at each other or playing FIFA on our Sony Playstation.
+
+As a Lanetix engineer for the past year and a half, I have grown by leaps and bounds.  We are looking for smart, motivated, driven and passionate engineers who want to challenge themselves and make an amazing product.  If you fit this description, feel bring to email us your CV and cover letter.  We would love to introduce you to the __best engineering team you will ever meet__!
```
Comment (apolishch) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: feel free, not just feel

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown (hunk 1 above)
Comment (apolishch) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: also I don't know if this is neccesarily supportive of "newest and best". Might make sense to mention Reflux and maybe even React, and drop Java.
Personally I'm not sure if I'd bother listing third party services (CircleCI, Intercom, etc), and would probably talk a lot more about Continuous Deployment with multiple daily ships to production

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+5. Our engineers are also leaders.  We encourage our team members to come up with new ideas to make our work more efficient, joyful and productive.  We share interesting articles and podcasts with each other.  We appreciate diversity and that is why many of our engineers mentor students at [Hackbright Academy](http://www.hackbrightacademy.com/), an engineering fellowship for women, and we hire some of the most promising graduates.
+
+6. Our engineers are not divided into front-end and back-end.  Each member is a full stack engineer.  We enjoy learning new skills and want to be able to grow and challenge ourselves.  The same is true for our use of programming languages; most of our code is written in NodeJS and Java.  If one of our engineers wants to learn new programming language, Lanetix is completely supportive.  In the future, I hope to learn [Scala](http://www.scala-lang.org/), and I feel lucky to work at a company that supports continuing education for engineers.   This is crucial in an ever growing and expanding technology market.
```
Comment (apolishch) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: I don't really know if saying "most of our code is written in NodeJS and Java" is accurate. How much Java do we actually have written? NodeJS and AngularJS seems to be much more honest, pertinent, and attractive imho

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+
+3. At Lanetix, we use only the best and newest engineering tools.  For example, we use [NodeJS](http://nodejs.org/), [Java](https://www.java.com/en/), [Bootstrap](http://getbootstrap.com/), [D3](http://d3js.org/), [AngularJS](https://angularjs.org/), [PostgreSQL](http://www.postgresql.org/), [Redis](http://redis.io/), [Elasticsearch](http://www.elasticsearch.org/) and [RabbitMQ](http://www.rabbitmq.com/).  We deploy our servers on [Heroku](https://www.heroku.com/)’s platform, and use [Cloudflare](https://www.cloudflare.com/) to manage our DNS.  We talk to our users using [Intercom](https://www.intercom.io/), and utilize [Pagerduty](http://www.pagerduty.com/) to make sure our servers are always up and running.  We track user errors using [Sentry](https://getsentry.com/), and store our code on [GitHub](https://github.com/).  We use [CircleCI](https://circleci.com/) andfor continuous integration and push our features and bug fixes to production up to several hundreds times per day.  Our skills and tools ensure that the company will move faster than our competitor, and provide the best experience and quality for our users.
```
Comment (apolishch) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: I don't know if this is neccesarily supportive of "newest and best". Might make sense to mention Reflux and maybe even React, and drop Java.
Personally I'm not sure if I'd bother listing third party services (CircleCI, Intercom, etc), and would probably talk a lot more about Continuous Deployment with multiple daily ships to production irrespective of the specific tools used

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown (hunk 3 above)
Comment (DannyDouglass) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: We should probably remove Java - the only thing written in Java is going to be deleted/replaced soon. We really only write in JavaScript at this time, but it is probably worth noting we like to pick the right technology for the task at hand. Just a thought...

**Comment by apolishch** [comment]
I would probably also mention how supportive the company is of open source contributions, both at work and at home. Imho this is a MAJOR selling point

File: _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+
+3. At Lanetix, we use only the best and newest engineering tools.  For example, we use [NodeJS](http://nodejs.org/), [Bootstrap](http://getbootstrap.com/), [D3](http://d3js.org/), [AngularJS](https://angularjs.org/), [ReflexJS](https://github.com/spoike/refluxjs), [PostgreSQL](http://www.postgresql.org/), [Redis](http://redis.io/), [Elasticsearch](http://www.elasticsearch.org/) and [RabbitMQ](http://www.rabbitmq.com/).  We deploy our servers on [Heroku](https://www.heroku.com/)’s platform, and use [Cloudflare](https://www.cloudflare.com/) to manage our DNS.  We talk to our users using [Intercom](https://www.intercom.io/), and utilize [Pagerduty](http://www.pagerduty.com/) to make sure our servers are always up and running.  We track user errors using [Sentry](https://getsentry.com/), and store our code on [GitHub](https://github.com/).  We use [CircleCI](https://circleci.com/) andfor continuous integration and push our features and bug fixes to production up to several hundreds times per day.  Our skills and tools ensure that the company will move faster than our competitor, and provide the best experience and quality for our users.
```
Comment (apechimp) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: don't forget io.js :stuck_out_tongue_closed_eyes:

Comment (OronNadiv) [inline _posts/2015-03-05-why-lanetix-has-the-best-engineering-team-you-will-ever-meet.markdown]: We don't (yet) use io.js.  We can have a separate blog post that shares our thoughts about io.js vs nodejs.

**Comment by OronNadiv** [comment]
@apolishch Let's have a separate blog post where we share our thoughts about open source.  This is a huge topic that has many different aspects.

**Pull request merged**